README.md

```diff
@@ -53,6 +65,24 @@ after(() => {
 
 ```
 
+### cy.setupPactHeaderAllowlist([header])
+Add a list of headers that will be included in a pact at test case level
```

Comment:
I think you should also mention here that it completely overrides any global defaults (i.e. it doesn't append to the globals, it sets them).